models/demos/deepseek_v3/tt/moe.py

```diff
@@ -50,36 +51,29 @@ def convert_weights(
         return weight_config
 
     @classmethod
-    def model_config(
+    def create_state(
         cls,
         hf_config: PretrainedConfig,
         mesh_device: ttnn.Device,
-        ccl: CCL1D,
         mode: str,
```

Comment:
state is shared between prefill and decode, so there should be no argument such as "mode"